citesphere/src/main/java/edu/asu/diging/citesphere/core/zotero/IZoteroManager.java

```diff
@@ -81,4 +81,7 @@ CitationResults getCollectionItems(IUser user, String groupId, String collection
     long getLatestGroupVersion(IUser user, String groupId);
         
     Map<ItemDeletionResponse, List<String>> deleteMultipleItems(IUser user, String groupId, List<String> citationKeys, Long citationVersion) throws ZoteroConnectionException, ZoteroHttpStatusException;
+    
+    ICitationCollection createCitationCollection(IUser user, String groupId, String collectionName, String parentCollection) throws ZoteroItemCreationFailedException, ZoteroConnectionException;
```

Comment:
needs javadoc (including explanation of parentCollection: null if no parent collection?).